docs/rules/function-component-definition.md

```diff
@@ -135,32 +135,6 @@ function getComponent () {
 
 ```
 
-## Unfixable patterns
```

Comment:
I deleted that since it is not possible to write:
```js
export default const getComponent = function () {
  return <div />;
}
```

but: 
```js
const getComponent = function () {
  return <div />;
}

export default getComponent
```

Reply:
Please restore this section; it’s not saying you’d write that code, it’s saying that because you can’t, the rule can’t autofix in certain configurations.